randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1705,26 +1705,46 @@ Extra - asset_id: collision_camera_019
   * Layers: default
   * Morph Ball Tunnel to Energy Recharge Station Secret/Tunnel to Energy Recharge Station (Gate)
   > Tile Group (WEIGHT)
-      After Ferenia - grapplepulloff1x2_000 and Lay Bomb
+      All of the following:
+          After Ferenia - grapplepulloff1x2_000
+          Lay Bomb or Lay Power Bomb
 
 > Bomb Ledge; Heals? False
   * Layers: default
   > Dock to EMMI Zone Exit Middle
       Screw Attack
   > Event - Grapple Block (grapplepulloff1x2_000)
       All of the following:
-          Morph Ball
+          Grapple Beam and Morph Ball
           Any of the following:
               Cross Bomb
               All of the following:
                   # Bomb & Power Bombs, to desync the tunnel bomb blocks from the ledge bomb blocks
                   Bomb and Power Bombs
+              All of the following:
+                  # Power bombs (video: https://twitter.com/i/status/1545111396760690692 )
+                  Power Bombs ≥ 2 and Movement (Beginner)
+              All of the following:
+                  # Bombs (video: https://twitter.com/SantosJuice/status/1545703077138665472 )
+                  Bomb and Movement (Intermediate)
   > Tile Group (WEIGHT)
-      Morph Ball
+      Trivial
   > Tunnel to Energy Recharge Station Secret
       All of the following:
-          Morph Ball and After Ferenia - grapplepulloff1x2_000
-          Cross Bomb or Power Bombs
+          After Ferenia - grapplepulloff1x2_000
+          Any of the following:
+              Any of the following:
+                  # Without flash shift or bombs
+                  Lay Cross Comb
```

Comment:
Wouldnt this also require movement, spin, or flash? wouldnt you just fall down to the weight block if you try to use cross to reach the tunnel?

Reply:
This is the last potential issue as far as i can see

Reply:
You could also move Morph Ball back to be with the event. Don't really know why you took out of always being needed. Can leave it there and just list Cross Bomb instead of Lay Cross Comb

Reply:
> You could also move Morph Ball back to be with the event. Don't really know why you took out of always being needed. Can leave it there and just list Cross Bomb instead of Lay Cross Comb

This is a nitpicky comment/review which does not hold any merit. "Lay Cross Comb" is currently not even used. For both the grapple event and tunnel morph ball is currently at the highest level. It is grouped with grapple beam for grapple event.
![image](https://user-images.githubusercontent.com/95079409/179690926-4e5c73d9-2d37-42ea-acb6-45807dad41fa.png)
And morphball is also grouped with after grapple event after 
![image](https://user-images.githubusercontent.com/95079409/179692432-f277d204-a6a2-4f73-a089-7ffcd39c19dc.png)

Did you forget to fetch the latest commit?

Reply:
this commit was not in yet when i commented